index.js

```diff
@@ -867,10 +879,13 @@ app.post('/smokeTest/teardown', async (request, response) => {
     await redis.addStateMachineData('Reset', 'SmokeTestLocation')
     response.status(200).send()
   } catch (error) {
-    helpers.log(`Smoke test setup error: ${error}`)
+    helpers.logError(`Smoke test setup error: ${error}`)
   }
 })
 
+// The error handler must be before any other error middleware and after all controllers
+app.use(Sentry.Handlers.errorHandler())
+
```

Comment:
Hmmm, this too would be nice to have in Brave Alert Lib, although by your comment here it should like maybe it's a function separate from `setupSentry`, maybe `setupSentryErrorHandler`. Funny requirement for its placement.